Sources/SPMBuildCore/BuildSystem/BuildSystem.swift

```diff
@@ -168,6 +168,14 @@ public struct BuildSystemProvider {
     }
 }
 
+extension BuildSystemProvider.Kind {
+    public var useXcodeBuildEngine: Bool {
```

Comment:
This should be spelled as `usesXcodeBuildEngine` to conform to the API Design Guidelines: https://www.swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage

> Uses of Boolean methods and properties should read as assertions about the receiver when the use is nonmutating, e.g. `x.isEmpty`, `line1.intersects(line2)`.

```suggestion
    public var usesXcodeBuildEngine: Bool {
```